index.js

```diff
@@ -386,7 +296,9 @@ function commandHandler(fullCommand, sendFunction, topic) { // If a status is se
                     sendFunction(topic, JSON.stringify({command: command, result: 'OK'}));
                     break;
                 case 'closetunnel':          // Close the SSH tunnel
-                    quipu.handle('closeTunnel');
+                    sshProcess.kill();
```

Comment:
This kill send a SIGTERM to the ssh process. I'm not sure it will close the tunnel correctly.

I think we should first try to kill it gently : 
```js
sshProcess.kill('SIGINT');
setTimeout(function () {
    if (sshProcess)
        sshProcess.kill();
}, 1000);
```